src/routes/index.js

```diff
@@ -0,0 +1,17 @@
+// import express
+const Router = require("express");
+
+// import views.js
+const viewRoutes = require("./viewRoutes");
+const apiRoutes = require("./api/api");
+
+// create router instance
+
+const router = Router();
+
+// register routes
+
+router.use("/api", apiRoutes);
```

Comment:
Add a `*` route to avoid the app breaking when entering a path other than `/` and `/notes`